tests/yaml_root/__snapshots__/jsfmt.spec.js.snap

```diff
@@ -282,9 +282,9 @@ pairs:
 set:
   # Explicitly typed set.
   baseball players: !!set
-    ? Mark McGwire
-    ? Sammy Sosa
-    ? Ken Griffey
+    Mark McGwire:
+    Sammy Sosa:
+    Ken Griffey:
```

Comment:
IMO we should keep the `?` syntax in this case (`!!set`) since these keys can’t have values and they can be any type of object.